src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -106,6 +187,16 @@
             """)
     List<Map<String, Object>> getCreatedResults(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select r.completionDate as day, count(r.id) as amount
+            from Result r
+            where r.completionDate >= :#{#startDate} and r.completionDate <= :#{#endDate} and r.participation.exercise.id in :exerciseIds
+            group by r.completionDate
+            order by r.completionDate
+            """)
+    List<Map<String, Object>> getCreatedResultsForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate,
```

Comment:
please use a custom object here and not a map

Reply:
Done